src/platform_impl/linux/mod.rs

```diff
@@ -854,12 +852,12 @@ impl<T> EventLoopWindowTarget<T> {
     }
 
     #[inline]
-    pub fn set_device_event_filter(&self, _filter: DeviceEventFilter) {
+    pub fn listen_device_events(&self, _filter: DeviceEvents) {
```

Comment:
should be `set_` something ?

Reply:
I don't think in this file that makes sense as it doesn't conflict with other function names.
Happy to change it if a maintainer thinks otherwise.